test/spectcl.js

```diff
@@ -348,16 +348,35 @@ describe('spectcl', function(){
         })
     })
 
+    describe('send', function(){
+        it('should call the cb once data has been written to child', function(done){
+            var session = new Spectcl()
+            session.spawn('node --interactive')
+            session.expect([
+                '>', function(){
+                    assert.notEqual(session.expect_out.match, null, 'null expect_out match')
+                    assert.notEqual(session.expect_out.buffer, '', 'empty expect_out buffer')
+                    session.send('process.exit()\r', function(){
+                        session.child.on('exit', function(){
+                            done()
+                        })
+                    })
+                }
+            ])
+        })
+    })
+
     describe('sendEof', function(){
         it('should kill the child process, emitting an exit event', function(done){
             var session = new Spectcl()
-            session.on('exit', function(){
```

Comment:
I will break this test into two.  The first will watch for exit and call `sendEof` with no callback, the other will call `sendEof` with a callback and call `done()` in the callback function.